offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -272,12 +271,12 @@ class CreatePersonView extends VerticalLayout {
                 createPersonViewModel.emailValid = true
             }
         })
-        this.addressAdditionComboBox.addSelectionListener({ selection ->
-            ValidationResult result = selectionValidator.apply(selection.getValue(), new ValueContext(this.addressAdditionComboBox))
+        this.organisationComboBox.addSelectionListener({ selection ->
+            ValidationResult result = selectionValidator.apply(selection.getValue(), new ValueContext(this.organisationComboBox))
             if (result.isError()) {
                 createPersonViewModel.affiliationValid = false
                 UserError error = new UserError(result.getErrorMessage())
-                addressAdditionComboBox.setComponentError(error)
+                organisationComboBox.setComponentError(error)
```

Comment:
I think there went something wrong during the merge. It should be the addressAdditionCombobox since it contains the affiliation and should determine if the affiliation is valid.

Reply:
Good point i created an issue #483